app/bundles/LeadBundle/Config/config.php

```diff
@@ -367,7 +369,7 @@
                 ],
             ],
             'mautic.lead.reportbundle.subscriber' => [
-                'class'     => 'Mautic\LeadBundle\EventListener\ReportSubscriber',
+                'class'     => ReportSubscriber::class,
```

Comment:
Let's use the full namespace here to keep consistency and makes it easier to read the config file (i.e. line 754)